definitions/redux_v3.x.x/flow_>=v0.23.x/redux_v3.x.x.js

```diff
@@ -0,0 +1,29 @@
+declare module 'redux' {
+  declare type State = any;
+  declare type Action = Object;
```

Comment:
I'm a noob on the nuances of `redux`, so a question: Does it make sense to define at least a `type: string` property here?

i.e. `{type: string, [_: string]: any}`